tests/integration/security/testdata/rbac/v1beta1-mtls.yaml.tmpl

```diff
@@ -0,0 +1,49 @@
+# Enforce access control based on mTLS identities.
```

Comment:
Could you add a description of what the policy does like how we do it in existing tests? Same with the other file.

Reply:
Done.